pyproject.toml

```diff
@@ -48,7 +48,7 @@ djangocms-blog = "1.1.1"
 djangocms-bootstrap4 = "2.0.0"
 djangocms-column = "1.11.0"
 djangocms-file = "3.0.0"
-djangocms-forms = "0.2.5"
+djangocms-forms-maintained = { git = "https://github.com/avryhof/djangocms-forms", rev = "30f464303d317cf7a3170ad64e11ab4471b4287b" }
```

Comment:
Wes, are you sure you can't use a tag? Is `30f4643…` a commit? I can't find it on [the repo's `master` branch](https://github.com/avryhof/djangocms-forms/commits/master).

Reply:
[you can manually add an entry into your pyproject.toml like this](https://github.com/python-poetry/poetry/issues/313#issuecomment-433184100)

Reply:
Well, there are [no recent tags](https://github.com/avryhof/djangocms-forms/tags). I must use a commit.

Reply:
Tested fix: 34f51d6